util/container/image_test.go

```diff
@@ -120,6 +120,22 @@ func TestParseImageName(t *testing.T) {
 			tag:       "",
 			wantError: true,
 		},
+		{
```

Comment:
Isn't this testing the wrong method? Shouldn't these cases be added to TestModifyImageTag?